### DISCUSSION TITLE
regular controller support

### DISCUSSION THREAD

**Discussion by curif** [curif/AgeOfJoy-2022.1#82] [Ideas]
Superluminal Goat said:
Hello, could you possibly add regular controller
support? Thank you

**Comment by WilsonBradley** [comment]
What is a regular controller?

**Reply by curif** [reply]
I don't know either. I should ask to Superluminal Goat...

This was a quick note to not forget the user suggestion.

**Reply by Elleby77** [reply]
I assume they mean a standard video game console controller hooked in via Bluetooth.      I've not done that yet with the Quest,  but as it was a feature that the Oculus Go could do,  I imagine the Quest series can as well.

**Reply by WilsonBradley** [reply]
Honestly, when people (original poster) are asking for help /enhancements , it's their responsibility to provide details to keep the world from assuming things. Else, I would just ignore it... Just food for thought..

**Reply by Elleby77** [reply]
Well controller support would likely be a good idea long term so that it can support more games.    However, I wouldn't want to overwhelm curif with too much.

**Reply by curif** [reply]
I had a dream where people made it's own physical cabinets, mounted controls such as joysticks and buttons and connected them to the headset using USB Zero Delay cards... however, it was only a dream...

**Comment by Elleby77** [comment]
I'd like to formally request Bluetooth controller (gamepad) support, if possible.

Using a controller would feel better for games that require more than 2 buttons.  I'm pretty sure Libretto supports them officially.

Plus, imagine instead of controlling it with the Quest's controllers you can use a Bluetooth arcade stick, like this one:  https://www.amazon.com/dp/B07D26RD9P/
